lib/tasks/router.rake

```diff
@@ -1,33 +0,0 @@
-namespace :router do
-  task :router_environment => :environment do
-    require 'plek'
-    require 'gds_api/router'
-
-    @router_api = GdsApi::Router.new(Plek.current.find('router-api'))
-  end
-
-  task :register_backend => :router_environment do
-    @router_api.add_backend('frontend', Plek.current.find('frontend', :force_http => true) + "/")
-  end
-
-  task :register_routes => :router_environment do
-    routes = [
-      %w(/ exact),
-      %w(/search exact),
```

Comment:
See my comment here https://github.com/alphagov/rummager/pull/480#issuecomment-129410826 - I think these search routes might be better being registered by `frontend`.